modules/running-distributed-data-science-workloads-from-notebooks.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads-from-notebooks_{context}"]
+= Running distributed data science workloads from notebooks
+
+[role='_abstract']
+To run a distributed data science workload from a notebook, you must first clone the `codeflare-sdk` repository, and provide the link to your Ray cluster image.
+
+.Prerequisites
+* You have created a data science project that contains a workbench that is running one of the default notebook images, for example, the *Standard Data Science* notebook.
+See the table in link:{rhodsdocshome}{default-format-url}/working_on_data_science_projects/creating-and-importing-notebooks_notebooks#notebook-images-for-data-scientists[Notebook images for data scientists] for a complete list of default notebook images.
+* You have launched your notebook server and logged in to Jupyter.
+
+.Procedure
+. [One-off step] If you have not already done so, clone the `codeflare-sdk` repository as follows:
```

Comment:
I was trying to clarify that you don't have to do this step every time you run a distributed workload from a notebook. You do it _once_, before you run your _first_ distributed workload from a notebook. 

I wondered whether it would be better to put it in the prereqs section, because it's not really part of the procedure -- only the first time. WDYT? Maybe I'm overthinking it, and it's fine to just remove the text as you suggested?

Reply:
I'll just remove the text as you suggested. If anyone is confused, I'll rewrite :)